deps/streamsearch/sbmh.js

```diff
@@ -101,18 +101,20 @@ SBMH.prototype._sbmh_feed = function(data) {
     while (pos < 0 && pos <= len - needle_len) {
```

Comment:
this while loop is now the biggest performance bottle neck. 

Maybe we optimize it some other day if it is possible.